public/app/ui/pages/home/home.html

```diff
@@ -17,27 +17,33 @@
         <div class="col-md-6 col-sm-6 col-xs-12 text-center">
             <div class="well">
                 <h2><a href="/photographs/search">Photographs Search</a></h2>
-                <p><a href="/photographs/search" class="btn btn-primary active" role="button">Access digitised photography</a></p>
+                <p><a href="/photographs/search" class="btn btn-primary" role="button">Access digitised photography</a></p>
             </div>
         </div>
         <div class="col-md-6 col-sm-6 col-xs-12 text-center">
             <div class="well">
                 <h2><a href="/library/search">Library Search</a></h2>
-                <p><a href="/library/search" class="btn btn-primary active" role="button">Access library catalogue</a></p>
+                <p><a href="/library/search" class="btn btn-primary" role="button">Access library catalogue</a></p>
             </div>
         </div>    
         <div class="col-md-6 col-sm-6 col-xs-12 text-center">
             <div class="well inner-left-well">
                 <h2><a href="/museum/search">Museum Search</a></h2>
-                <p><a href="/museum/search" class="btn btn-primary active" role="button">Access museum collection</a></p>
+                <p><a href="/museum/search" class="btn btn-primary" role="button">Access museum collection</a></p>
             </div>
         </div>
         <div class="col-md-6 col-sm-6 col-xs-12 text-center">
             <div class="well inner-right-well">
                 <h2><a href="/memorials/search">Public Memorials Search</a></h2>
-                <p><a href="/memorials/search" class="btn btn-primary active" role="button">Access memorial records</a></p>
+                <p><a href="/memorials/search" class="btn btn-primary" role="button">Access memorial records</a></p>
             </div>
-        </div>  
+        </div>
+        <div class="hidden-lg hidden-md hidden-sm col-xs-12 text-center">
```

Comment:
In mobile view, I had to hide the search bar in the header, so I've added this here.  This is hidden in the other views because it breaks the 2x2 layout and is redundant.